samples/ReadMe/Provider.Tests/TestStartup.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace ReadMe.Provider.Tests
+{
+    public class TestStartup
+    {
+        private readonly Startup inner;
+
+        public TestStartup(IConfiguration configuration)
+        {
+            inner = new Startup(configuration);
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            inner.ConfigureServices(services);
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            app.UseMiddleware<ProviderStateMiddleware>()
+                .UseMiddleware<AuthorizationTokenMiddleware>();
```

Comment:
I wonder if it might be slightly overkill adding auth to this simple example. It adds a fair bit of semi-complex code which doesn't really relate to the actual core example of how to write Pact tests, whereas the other samples cover auth as they're more broadly scoped.

Reply:
Let's see if I can pull out all the auth code and still make it work. Agreed that it adds a lot of complexity and no value to the example.